.github/pull_request_template.md

```diff
@@ -1,26 +1,25 @@
-<!-- Mark items with [x] where applicable -->
+<!-- Uncomment relevant sections and delete the lines which are not applicable -->
 
-#### General
-- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)
-
-#### Have the results of the proposed changes been tested?
-- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
-- [ ] I generally don't use the affected packages but briefly tested this PR
+<!-- Note: PRs should only have one entry in the following section, otherwise
+it will be assumed they have not been tested at all. -->
```

Comment:
Is this fine or is it too harsh?

Reply:
Policy is just right, statement is kind of implied by previous one.

Reply:
not harsh and to the point.

how about something like `*YES*|*NO*` (in bolds) to make it both (even more) obvious to contributor that an answer is required and obvious to maintainer if it wasn't answered (like not checking any checkbox before).